### PR TITLE
F126 orwdxa isactoi

### PR DESCRIPTION
**Pull request by sidjtd**
My first time dealing with dates! Thanks to Moment.  Didn't care for the redundant return statement at the very end of processRPCInvocation but the airbnb linter was annoying so... it's there.

### CODE REVIEW

File: prototypes/files/f126_orwdxaIsactoi.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                example: 156,
+            },
+        ],
+        processRPCInvocation(rpcArgs) {
+            let vdmObj = {};
+
+            try {
+                // If VDM obj exists, assign it. Else, return ''
+                vdmObj = VDM.describe(`101_43-${rpcArgs[0]}`);
+
```
Comment (EdisonTing) [inline prototypes/files/f126_orwdxaIsactoi.js]: I suggest move other codes outside of try and catch block.

Comment (sidjtd) [inline prototypes/files/f126_orwdxaIsactoi.js]: Did IT!

File: prototypes/files/f126_orwdxaIsactoi.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            try {
+                // If VDM obj exists, assign it. Else, return ''
+                vdmObj = VDM.describe(`101_43-${rpcArgs[0]}`);
+            } catch (e) {
+                return '';
+            }
+
+            // If obj has key, proceed. Else, return ''
+            const retrievedDate = vdmObj.inactivated.value;
+            if (_.isEmpty(retrievedDate)) return '';
```
Comment (EdisonTing) [inline prototypes/files/f126_orwdxaIsactoi.js]: minor thing. You can move that if logic to line 47. if (!_.isEmpty(...) && (moment...))

Comment (mfuroyama) [inline prototypes/files/f126_orwdxaIsactoi.js]: also consider using `_.get`, but sounds like you have this well in hand

**Pull request merged**